src/picker_gui.cpp

```diff
@@ -398,6 +403,18 @@ void PickerWindow::OnClick(Point pt, WidgetID widget, int)
 			this->InvalidateData({PickerInvalidation::Class, PickerInvalidation::Type, PickerInvalidation::Position});
 			break;
 
+		case WID_PW_SHRINK:
+			this->callbacks.preview_height = this->preview_height = std::max(PREVIEW_HEIGHT, this->preview_height - 16);
```

Comment:
Constant? (Here and for the expand button)

```suggestion
			this->callbacks.preview_height = this->preview_height = std::max(PREVIEW_HEIGHT, this->preview_height - PREVIEW_HEIGHT_STEP);
```